x-pack/plugins/ml/public/application/management/jobs_list/components/jobs_list_page/space_management/space_management.tsx

```diff
@@ -101,12 +102,19 @@ export const SpaceManagement: FC<Props> = ({ spacesApi, setCurrentTab }) => {
     [getList, loadingTab]
   );
 
+  useEffect(() => {
+    onReload(() => () => refresh(currentTabId));
```

Comment:
Is there a reason to have a callback that returns a callback? Also, this effect mainly depends on the `currentTabId`. I reckon this logic can be removed and a parent component can handle this just by using the `onTabChange` callback.